.github/workflows/ci.yml

```diff
@@ -104,3 +104,32 @@ jobs:
         run: |
           tensorflow/lite/micro/tools/ci_build/test_makefile.sh
           tensorflow/lite/micro/tools/ci_build/test_x86.sh
+
+  label_cleanup:
+    runs-on: ubuntu-latest
+
+    if: |
+      (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'ci:run')) ||
+      (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'ci:test'))
+
+    name: remove CI runner labels
+    steps:
+      - name: remove tags
+        uses: actions/github-script@a3e7071a34d7e1f219a8a4de9a5e0a34d1ee1293
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
will GITHUB_TOKEN work if the PR is coming from a fork?

use the tflm bot repo token instead?
https://github.com/tensorflow/tflite-micro/blob/0afb62f9d1074df11c92f259a2eb1fc936009b96/.github/workflows/sync.yml#L51

Reply:
Bah, yeah, the trigger is going to have to be changed to pull_request_target. Pretty sure that is necessary for either one to work, since we haven't been able to get at secrets from fork PRs without that.

Reply:
ah yes, you're right.